libxrdp/xrdp_caps.c

```diff
@@ -901,19 +922,29 @@ xrdp_caps_send_demand_active(struct xrdp_rdp *self)
     out_uint16_le(s, 5);
     out_uint8(s, 0); /* client sets */
 
-    if (self->client_info.use_fast_path & 1) /* fastpath output on */
+    if (self->client_info.use_fast_path & FASTPATH_OUTPUT_SUPPORTED) /* fastpath output on */
```

Comment:
Great! The comment is not very useful now, it can be removed or replaced with something deeper.